src/content/app/genome-browser/hooks/useGenomeBrowserIds.ts

```diff
@@ -41,68 +31,18 @@ import {
 const useGenomeBrowserIds = () => {
   const activeGenomeId = useAppSelector(getBrowserActiveGenomeId);
   const activeFocusObjectId = useAppSelector(getBrowserActiveFocusObjectId);
-  const committedSpecies = useAppSelector((state) =>
-    getCommittedSpeciesById(state, activeGenomeId ?? '')
-  );
-  const params = useUrlParams<'genomeId'>('/genome-browser/:genomeId');
-  const { search } = useLocation();
-
-  const { genomeId: genomeIdInUrl } = params;
-  const urlSearchParams = new URLSearchParams(search);
-  const focusObjectIdInUrl = urlSearchParams.get('focus');
-
-  const {
-    currentData: genomeInfo,
-    isFetching,
-    isError,
-    error
-  } = useGenomeInfoQuery(genomeIdInUrl ?? '', {
-    skip: !genomeIdInUrl
-  });
-  const genomeId = genomeInfo?.genomeId;
-  const isMissingGenomeId = isError && isGenomeNotFoundError(error);
-
-  // TODO: check if the logic below is correct
-  const genomeIdForUrl =
-    genomeIdInUrl ??
-    committedSpecies?.genome_tag ??
-    committedSpecies?.genome_id;
-
-  let focusObjectId;
-  let focusObjectIdForUrl;
-  let parsedFocusObjectId;
-  let isMalformedFocusObjectId = false;
+  const genomeBrowserIdsContext = useContext(GenomeBrowserIdsContext);
 
-  if (focusObjectIdInUrl) {
-    focusObjectIdForUrl = focusObjectIdInUrl;
-    if (genomeId) {
-      try {
-        parsedFocusObjectId = {
-          genomeId,
-          ...parseFocusIdFromUrl(focusObjectIdInUrl)
-        };
-        focusObjectId = buildFocusObjectId(parsedFocusObjectId);
-      } catch {
-        isMalformedFocusObjectId = true;
-      }
-    }
-  } else if (activeFocusObjectId) {
-    focusObjectIdForUrl = buildFocusIdForUrl(activeFocusObjectId);
+  if (!genomeBrowserIdsContext) {
+    throw new Error(
+      'useGenomeBrowserIds must be used with GenomeBrowserIdsContext Provider'
+    );
   }
 
   return {
-    isFetchingGenomeId: isFetching,
-    isMissingGenomeId,
-    genomeId,
-    genomeIdInUrl,
-    focusObjectId,
-    focusObjectIdInUrl,
+    ...genomeBrowserIdsContext,
     activeGenomeId,
-    activeFocusObjectId,
-    genomeIdForUrl,
-    focusObjectIdForUrl,
-    parsedFocusObjectId,
-    isMalformedFocusObjectId
+    activeFocusObjectId
```

Comment:
`activeFocusObjectId` is available in GenomeBrowserIdsContextProvider. Why not add it to the context there?